sdk/network/Azure.ResourceManager.Network/src/Generated/PacketCaptureData.cs

```diff
@@ -106,6 +110,10 @@ internal PacketCaptureData(ResourceIdentifier id, string name, ResourceType reso
         public PacketCaptureStorageLocation StorageLocation { get; }
         /// <summary> A list of packet capture filters. </summary>
         public IReadOnlyList<PacketCaptureFilter> Filters { get; }
+        /// <summary> This continuous capture is a nullable boolean, which can hold 'null', 'true' or 'false' value. If we do not pass this parameter, it would be consider as 'null', default value is 'null'. </summary>
+        public bool? ContinuousCapture { get; }
```

Comment:
Consider a cleaner boolean name like `IsCaptureContinous`

Reply:
Hi @archerzz , it's a nullable boolean parameter.